lib/src/widgets/bottom_bar_with_sheet_item.dart

```diff
@@ -44,12 +50,29 @@ class BottomBarWithSheetItemWidget extends StatelessWidget {
     if (model.label != null) {
       items.addAll(
         [
-          SizedBox(height: 2),
-          Text(
-            model.label!,
-            style:
-                isSelected ? theme.selectedItemTextStyle : theme.itemTextStyle,
-          ),
+          theme.isVerticalItemLabel
+              ? SizedBox(
+                  height: 2,
+                )
+              : SizedBox(
+                  width: 2,
+                ),
+          theme.isVerticalItemLabel
+              ? Text(
+                  model.label!,
+                  style: isSelected
+                      ? theme.selectedItemTextStyle
+                      : theme.itemTextStyle,
+                )
+              : Expanded(
+                  flex: 2,
+                  child: Text(
+                    model.label!,
+                    style: isSelected
+                        ? theme.selectedItemTextStyle
+                        : theme.itemTextStyle,
+                  ),
+                ),
```

Comment:
Please use the [spread operator](https://medium.com/flutter-community/whats-new-in-dart-2-3-1a7050e2408d#:~:text=1.-,The%20Spread%20operator,-(%E2%80%A6)) to the list

```dart
if(theme.isVerticalItemLabel)...[
SizedBox(height: 2,),
Text(
    model.label!,
    style: isSelected ? theme.selectedItemTextStyle : theme.itemTextStyle,
),]
else ...[
```